File: app/src/main/java/net/nashihara/naroureader/activities/SettingsActivity.java
package net.nashihara.naroureader.activities;

import android.databinding.DataBindingUtil;
import android.graphics.Color;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.balysv.materialmenu.MaterialMenuDrawable;

import net.nashihara.naroureader.R;
import net.nashihara.naroureader.databinding.ActivitySettingsBinding;
import net.nashihara.naroureader.fragments.SettingsFragment;

public class SettingsActivity extends AppCompatActivity {

    private ActivitySettingsBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_settings);

        MaterialMenuDrawable materialMenu = new MaterialMenuDrawable(this, Color.WHITE, MaterialMenuDrawable.Stroke.THIN);
        materialMenu.animateIconState(MaterialMenuDrawable.IconState.X);
        binding.toolbar.setTitle("設定");
        binding.toolbar.setNavigationIcon(materialMenu);
        binding.toolbar.setNavigationOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

        getSupportFragmentManager().beginTransaction()
                .replace(R.id.settings_content, new SettingsFragment())
                .commit();
    }
}

File: app/src/main/java/net/nashihara/naroureader/activities/SettingsActivity.kt
package net.nashihara.naroureader.activities

import android.databinding.DataBindingUtil
import android.graphics.Color
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.balysv.materialmenu.MaterialMenuDrawable

import net.nashihara.naroureader.R
import net.nashihara.naroureader.databinding.ActivitySettingsBinding
import net.nashihara.naroureader.fragments.SettingsFragment

class SettingsActivity : AppCompatActivity() {

    private lateinit var binding: ActivitySettingsBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView<ActivitySettingsBinding>(this, R.layout.activity_settings)

        val materialMenu = MaterialMenuDrawable(this, Color.WHITE, MaterialMenuDrawable.Stroke.THIN)
        materialMenu.animateIconState(MaterialMenuDrawable.IconState.X)
        binding.toolbar.title = "設定"
        binding.toolbar.navigationIcon = materialMenu
        binding.toolbar.setNavigationOnClickListener { finish() }

        supportFragmentManager.beginTransaction()
                .replace(R.id.settings_content, SettingsFragment())
                .commit()
    }
}
